lib/knapsack_pro/test_file_finder.rb

```diff
@@ -63,6 +63,10 @@ def test_files
         return KnapsackPro::Config::Env.test_file_list.split(',').map(&:strip)
       end
 
+      if test_file_list_enabled && KnapsackPro::Config::Env.test_file_list_source_file
+        return File.read(ENV['KNAPSACK_PRO_TEST_FILE_LIST_SOURCE_FILE']).split(/\n/)
```

Comment:
Could you replace `ENV['KNAPSACK_PRO_TEST_FILE_LIST_SOURCE_FILE']` with `KnapsackPro::Config::Env.test_file_list_source_file`?

We have hidden env var name in method `KnapsackPro::Config::Env.test_file_list_source_file` so let's use it.

Reply:
I'm wondering if we should worry about handling the case when someone has a new line at the very end of the source file.
Will `.split(/\n/)` handle it? 

I can imagine a developer having an IDE that will auto add a new line and breaks this. Or maybe someone will generate a source file with new line at the end. 

We could cover the case in spec when the source file has a new line at the end.

Reply:
yes, the `.split(/\n/)` can handle the new line in the end. I have added new lines in the testing file.

Reply:
github shows that there is no new line at the end of the `spec_fake/fake_test` file.

Reply:
I will push in mins